src/main/java/mate/academy/spring/service/impl/AuthenticationServiceImpl.java

```diff
@@ -28,7 +28,7 @@ public User register(String email, String password) {
         User user = new User();
         user.setEmail(email);
         user.setPassword(password);
-        user.setRoles(Set.of(roleService.getRoleByName(RoleName.USER)));
+        user.setRoles(Set.of(roleService.getByName(Role.RoleName.USER.name())));
```

Comment:
did u test it ? does this fix ur problem ? try to post request via Postman to /register

Reply:
i tested it, now it is working correct)